src/nft/components/explore/Banner.tsx

```diff
@@ -56,30 +58,36 @@ const Banner = () => {
       {collections && collections[current] ? (
         <Link to={`/nfts/collection/${collections[current].address}`} style={{ textDecoration: 'none' }}>
           <Box style={{ height: '386px' }}>
-            <CollectionWrapper bannerImageUrl={collections[current].bannerImageUrl}>
-              <Box
-                as="section"
-                className={section}
-                display="flex"
-                flexDirection="row"
-                flexWrap="nowrap"
-                paddingTop="40"
-                position="relative"
-              >
-                <CollectionDetails collection={collections[current]} hovered={hovered} rank={current + 1} />
-              </Box>
-              <Center marginTop="16" position="relative">
+            <div
+              className={styles.bannerWrap}
+              style={{ backgroundImage: `url(${collections[current].bannerImageUrl})` }}
+            >
+              <Box className={styles.bannerOverlay} width="full" />
+              <CollectionDetails collection={collections[current]} hovered={hovered} rank={current + 1} />
+              <Center marginTop="32" position="relative">
```

Comment:
Can you refactor this whole element out as `CarouselIndicator`?

Reply:
The moved out the individual indicators, but am still leaving the `.map` in the Banner component based on other feedback above.